atlas/responder/README.md

```diff
@@ -0,0 +1,53 @@
+# responder
```

Comment:
all readme file are template.
you can add short description for all services and remove unnecessary info